tinywallet/tinywallet/screens.py

```diff
@@ -241,12 +244,11 @@ def pop_(self, screen=None):
             return
         popped.setVisible(False)
         self.layout.removeWidget(popped)
-        # log.debug("pop setting top screen to %s" % type(top).__name__)
+        if hasattr(popped, "unstacked"):
+            popped.unstacked()
```

Comment:
This could be replaced by one line:
```getattr(popped, "unstacked", lambda: None)()```
Maybe it's less readable though.

Reply:
I'm gonna say it's a toss up and leave it.

Reply:
Actually, I did change this by implementing `pass`ing `stacked` and `unstacked` methods on the `Screen` base class. No need to check `hasattr` at all this way.